package.json

```diff
@@ -1,22 +1,39 @@
 {
   "name": "@qld-gov-au/qgds-tokens",
-  "version": "1.2.0",
+  "version": "2.0.0",
   "description": "This repository is configured to sync tokens via Figma.",
-  "main": "index.js",
+  "type": "module",
```

Comment:
please fix up the build.js file to be in module imports.
https://github.com/qld-gov-au/qgds-tokens/blob/feature/QGDS-243-multi-dimensional-theming/build.js#L1